### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by F0r3n5iC-Sw33t13**
Added some dates in README.md by the version numbers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,15 +57,15 @@ Fork this repo, contribute and help keep it alive
 Small update to fix a security issue
 
 
-#### 2.8.5.4
+#### 2.8.5.4 - 3th of September 2021
```
Comment (JohnVillalovos) [inline README.md:60]: `3th` is not a thing.  `3rd` would be correct.

But personally I would do dates like: 2021-09-03 or 03-Sep-2021

Comment (effgarces) [reply README.md:60]: Agree, in fact I would argue for using 2021-09-03, as it's probably the most universal version

Comment (F0r3n5iC-Sw33t13) [reply README.md:60]: Agreed, I will change it!

**Comment by colisee** [comment]
Hi @F0r3n5iC-Sw33t13 ,

I agree with your idea of implementing a TODO list, but not through file `README.md`, as per your commit 975d656

I think that @effgarces should rather use the **Projects** feature of github...

**Comment by F0r3n5iC-Sw33t13** [comment]
Hi @colisee, 

Sorry if it wasn't the right place but I did not know where I had to put it.

**Comment by F0r3n5iC-Sw33t13** [comment]
Hi!

Happy new year

@effgarces, is it okay with you if you gives me more right on this so we can make this happen?

**Pull request merged**